cli/pull.ml

```diff
@@ -1,3 +1,4 @@
+module Buenzli_fmt = Fmt
```

Comment:
Why is this rename necessary? If it is, how about something more neutral like `Fmt_lib`?

Reply:
Something more neutral would work indeed. The rename is need because `Stdune` as an `Fmt` module which is basically `Fmt` shrinked to the bare minimum.